update-config-files.php

```diff
@@ -0,0 +1,363 @@
+#!/usr/local/bin/php
+<?php
+define('VAR_NAME', 0); //var name
+define('VAR_VALUE', 1); //var value
+define('VAR_COMMENT', 2); //var comment
+define('VAR_TYPE', 3); // static or const
+define('NEW_LINE', "\n"); // new line
+define('TAB', "\t \t "); // Tab
+define('WORD_WRAP', 120); // word wrap
+
+/**
+ * [(php update-config-files.php -help) rertuns help]
+ * [(php update-config-files.php write-portal) writes all Portal db-configs in a dir]
+ * [(php update-config-files.php write-nexus) writes all Nexus configs in a dir]
+ */
+for ($i = 0; $i <= $argc - 1; $i++)
+{
+    switch ($argv[$i]) {
+    case '-help':
+      print "Note: Before running this file edit the array for add/remove of variables \n1. (write-portal) Writes Portal Config \n2. (write-nexus) Writes Nexus Config \n";
+
+    break;
+    case 'write-portal':
+      writePortalConfig();
+
+    break;
+
+    case 'write-nexus':
+      writeNexusConfig();
+
+    break;
+
+    default:
+      break;
+  }
+}
+
+  /**
+   * [writePortalConfig writes/backup all Portal db-configs in a root dir]
+   * To add a variable to the db-config files add it to the array or to remove a variable remove it from the array
+   * Note: if no value is supplied for a variable it will use the one in config file if exists
+   * @return [string] [config content]
+   */
+    function writePortalConfig()
+    {
+        $portalConfig = array(
+          'dbHost' => array('varName' => 'dbHost', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbName' => array('varName' => 'dbName', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbUser' => array('varName' => 'dbUser', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbPass' => array('varName' => 'dbPass', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+
+          'title' => array('varName' => 'title', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'city' => array('varName' => 'city', 'varValue' => '', 'varComment' => ''),
+          'adminLogonName' => array('varName' => 'adminLogonName', 'varValue' => '', 'varComment' => 'Administrator\'s logon name', 'varType' => ''),
+          'adPath' => array('varName' => 'adPath', 'varValue' => '', 'varComment' => 'Active directory path', 'varType' => ''),
+          'uploadDir' => array('varName' => 'uploadDir', 'varValue' => '', 'varComment' => 'Directory for user uploads. using backslashes (/), with trailing slash', 'varType' => ''),
+          'leafSecure' => array('varName' => 'leafSecure', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'orgchartPath' => array('varName' => 'orgchartPath', 'varValue' => '', 'varComment' => 'HTTP Path to orgchart with no trailing slash', 'varType' => ''),
+          'orgchartImportTags' => array('varName' => 'orgchartImportTags', 'varValue' => "array('')", 'varComment' => 'Import org chart groups if they match these tags', 'varType' => ''),
```

Comment:
`varValue` should be defaulted to empty string, so that it doesn't set `orgchartImportTags` to `array('')`